<?xml version="1.0" ?>
<launch>
    <arg name="x" default="8"/>
    <arg name="y" default="8"/>
    <arg name="ctrl_topic" default="/controller/control_point"/>
    <arg name="feedback" default="/formation/centroid"/>

    <param name="/map/configuration_map" type="string" value="$(find forma3_map)/saved_maps/configuration_space/room.pgm" />
    <param name="/map/free_space_map" type="string" value="$(find forma3_map)/saved_maps/free_space/room.pgm" />
    <param name="/map/voxel_size" type="double" value="0.05" />

    <param name="prm_divisions" type="int" value="25" />
    <param name="prm_nodes_to_try" type="int" value="12" />
    <param name="prm_hookup_dist" type="double" value="0.5" />

    <param name="local_motion_package" type="string" value="local_motion"/>
    <param name="local_motion_exe" type="string" value="bug2.py"/>
    <param name="local_motion_extra_args" type="string" value=""/>
    <param name="local_motion_eval" type="string" value="eval_length.py"/>

    <param name="control_point_topic" type="string" value="$(arg ctrl_topic)"/>
    <param name="feedback_topic" type="string" value="$(arg feedback)"/>
    <param name="control_tolerance" type="double" value=".2"/>

    <node name="prm_planner" pkg="planner" type="prm_planner.py" output="screen"
          args="$(arg x) $(arg y)" cwd="node" required="true"/>

    <!-- <group if="$(arg eval)">
        <node name="wavefront_eval" pkg="local_motion" type="eval_length.py" output="screen" cwd="node"/>
    </group> -->
</launch>
